This patch was prepared by Gwenole Beauchesne <gbeauchesne@mandrakesoft.com> for Mandrake
It enables use of the distribution's freetype library
--- oo_1.0_src/vcl/prj/build.lst.system-freetype	2002-02-20 13:20:38.000000000 +0100
+++ oo_1.0_src/vcl/prj/build.lst	2002-05-06 11:36:55.000000000 +0200
@@ -1,4 +1,4 @@
-vc	vcl	:	nas freetype psprint rsc sot ucbhelper unotools sysui NULL
+vc	vcl	:	nas psprint rsc sot ucbhelper unotools sysui NULL
 vc	vcl										usr1	-	all	vc_mkout NULL
 vc	vcl\source\unotypes						nmake	-	all	vc_unot NULL
 vc	vcl\source\glyphs						nmake	-	all	vc_glyphs vc_unot NULL
--- oo_1.0_src/vcl/source/glyphs/makefile.mk.system-freetype	2001-08-02 19:16:20.000000000 +0200
+++ oo_1.0_src/vcl/source/glyphs/makefile.mk	2002-05-06 13:27:50.000000000 +0200
@@ -64,6 +64,7 @@ PRJ=..$/..
 
 PRJNAME=vcl
 TARGET=glyphs
+INCPRE += -I/usr/include/freetype2
 
 # --- Settings -----------------------------------------------------
 
--- oo_1.0_src/vcl/source/glyphs/gcach_ftyp.cxx.system-freetype	2002-01-02 11:38:06.000000000 +0100
+++ oo_1.0_src/vcl/source/glyphs/gcach_ftyp.cxx	2002-05-06 13:31:49.000000000 +0200
@@ -77,9 +77,6 @@
 #include <osl/file.hxx>
 #include <osl/thread.hxx>
 
-// VERSION_MINOR in freetype.h is too coarse, we need to fine-tune ourselves:
-#define FTVERSION 205
-
 #include "freetype/freetype.h"
 #include "freetype/ftglyph.h"
 #include "freetype/ftoutln.h"
@@ -87,6 +84,8 @@
 #include "freetype/tttags.h"
 #include "freetype/ttnameid.h"
 
+#define FTVERSION ((FREETYPE_MAJOR * 100) + (FREETYPE_MINOR * 10) + FREETYPE_PATCH)
+
 #include <vector>
 
 // TODO: move file mapping stuff to OSL
--- oo_1.0_src/vcl/util/makefile.mk.system-freetype	2002-02-15 12:44:54.000000000 +0100
+++ oo_1.0_src/vcl/util/makefile.mk	2002-05-06 11:36:55.000000000 +0200
@@ -198,7 +198,7 @@ LIB1FILES+= \
 
 .IF "$(USE_BUILTIN_RASTERIZER)"!=""
     LIB1FILES +=    $(SLB)$/glyphs.lib
-    SHL1STDLIBS+=   $(FREETYPELIBST)
+    SHL1STDLIBS+=   $(FREETYPELIB)
 .ENDIF
 
 .IF "$(remote)" == ""
